src/main/java/seedu/duke/manager/OrderManager.java

```diff
@@ -0,0 +1,72 @@
+package seedu.duke.manager;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import seedu.duke.entities.Order;
+import seedu.duke.entities.Dish;
+
+
+public class OrderManager {
+    private ArrayList<Order> orders;
+    private DishManager dishManager = new DishManager();
+    private ArrayList<Dish> dishes;
+
+    public OrderManager() {
+        this.orders = new ArrayList<Order>();
+        this.dishes = dishManager.getDishes();
+    }
+
+    public void addOrder(Order order) {
+        this.orders.add(order);
+    }
+
+    public void addDishToOrder() {
+        Order order = null;
+        boolean notQuit = true;
+        System.out.println("Enter dishes you want to order (Q/q to exit): ");
+        Scanner sc = new Scanner(System.in);
```

Comment:
User input should be processed in the controller class

Reply:
> I think user input should be handled in the controller class

Same thought here.